PMMain/Midas/MidasGUI/MidasMain.cpp

```diff
@@ -80,11 +82,12 @@ int midasMain(MidasThread *threadHandle, SequenceDisplayer* sequenceDisplayer, I
     MyoDevice* myoDevice = new MyoDevice(&sharedData, &controlState, "com.midas.midas-test", sequenceDisplayer, infoIndicator);
     MouseCtrl* mouseCtrl = new MouseCtrl();
     KybrdCtrl* kybrdCtrl = new KybrdCtrl();
+    vector<ringData> kybrdRingData = threadHandle->readKeyboardSetupFile();
 
```

Comment:
Why is this calling the function from threadHandle? It shouldnt need to be a function of MidasThread